perceval/rendering/pdisplay.py

```diff
@@ -327,6 +327,22 @@ def pdisplay_tomography_chi(qpt: ProcessTomography, output_format: Format = Form
     plt.show()
 
 
+def pdisplay_graph(g: nx.Graph, output_format: Format = Format.MPLOT):
+    if output_format not in {Format.MPLOT, Format.LATEX}:
+        raise TypeError(f"Graph plot does not support {output_format}")
+    if output_format == Format.LATEX:
+        print(nx.to_latex(g))
```

Comment:
I think this should be `return nx.to_latex(g)`.
The higher layers of pdisplay will do the display.